frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh

```diff
@@ -47,29 +30,37 @@ data:
   volumeMode: Filesystem
 EOF
 
-# Enable live-migration feature-gate
-oc create -f - <<EOF
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: kubevirt-config
-  namespace: kubevirt
-  labels:
-  kubevirt.io: ""
-data:
-  feature-gates: "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar,Snapshot"
-EOF
+num_of_lines=0
+
+# Wait for templates to be created before continuing
+while [ "$num_of_lines" -eq 0 ]
+do
+  echo "Waiting for creation of common templates"
+  num_of_lines=$(oc get -n openshift templates | grep windows10-desktop-large | wc -l)
+done
+
+echo "Common templates created"
+
+# Pause the SSP operator to prevent overwrites
+oc annotate -n kubevirt-hyperconverged deployments ssp-operator kubevirt.io/operator.paused="true"
+
+# Annotate templates
+curl https://raw.githubusercontent.com/openshift/console/master/frontend/packages/kubevirt-plugin/integration-tests/ci-scripts/add-support-annotations.sh | bash
+
+# Unpause the SSP operator
+oc annotate -n kubevirt-hyperconverged deployments ssp-operator kubevirt.io/operator.paused-
 
-# TODO remove this once the test image is built from Dockerfile
 if ! type virtctl; then
-  # Install virtctl binary and add in to PATH
+  # Install virtctl binary and add to PATH
   mkdir virtctl
 
   wget ${VIRTCTL_AMD64} -O virtctl/virtctl || wget ${VIRTCTL_X86_64} -O virtctl/virtctl
   [[ ! -f "virtctl/virtctl" ]] && echo "ERROR: virtctl binary is unavailable for download" && exit 1
 
   chmod +x virtctl/virtctl
 
-  export PATH="${PATH}:$(pwd)/virtctl"
+  export PATH="${PATH}:$(pwd)/virtctl/virtctl"
```

Comment:
hmm, are you sure ?

Reply:
During my testing, it didn't recognize it without adding that. I saw the error that virtctl couldn't be found, though. I don't know why adding something to the path would be different in CI vs other methods, but I'll change it back. I found it odd that I had to change it in the first place since we've been using this in the current deploy script for some time now.